includes/Generator/Product.php

```diff
@@ -314,6 +315,7 @@ protected static function generate_variable_product() {
 			'image_id'          => self::get_image(),
 			'category_ids'      => self::get_term_ids( 'product_cat', self::$faker->numberBetween( 0, 3 ) ),
 			'tag_ids'           => self::get_term_ids( 'product_tag', self::$faker->numberBetween( 0, 5 ) ),
+			'brand_ids'         => self::get_term_ids( 'product_brand', 1),
```

Comment:
I couldn't get this to add generated brands. I had to use `wp_set_object_terms` in `generate`. 
It would also be best to assign a variable number for brands (as in category and tags).

```
// Assign brand terms using wp_set_object_terms
$brand_ids = self::get_term_ids( 'product_brand', self::$faker->numberBetween( 1, 3 ) );
if ( ! empty( $brand_ids ) ) {
	wp_set_object_terms( $product->get_id(), $brand_ids, 'product_brand' );
}
```